packages/nuxt/src/module.ts

```diff
@@ -12,15 +15,37 @@ export interface ModuleOptions {
    * @see https://onyx.schwarz/development/#installation
    */
   disableGlobalStyles?: boolean;
+  /**
+   * This module will automatically detect if @nuxtjs/i18n is used within the project and configure itself to work with it.
+   * This group of options gives you the flexibility to fine tune this integration.
+   */
+  i18n?: {
+    /**
+     * Prefix to use for all translation keys comming from onyx. The modules translations can later be used under this prefix, e.g. `$t('onyx.optional')`.
+     * @default "onyx"
+     */
+    prefix?: string;
+  };
 }
 
 export default defineNuxtModule<ModuleOptions>({
   meta: {
-    name: "onyx-nuxt",
+    name: "@sit-onyx/nuxt",
     configKey: "onyx",
   },
-  defaults: {},
+  defaults: {
+    i18n: { prefix: "onyx" },
+  },
   setup(options, nuxt) {
+    const logger = useLogger("@sit-onyx/nuxt");
+    const { resolve } = createResolver(import.meta.url);
+
+    nuxt.options.runtimeConfig.public = defu(nuxt.options.runtimeConfig.public, {
+      onyx: {
+        i18n: options.i18n,
+      },
```

Comment:
Would prefer not to have a new dependency introduced for this. We already have `deepmerge-ts`, which seems to serve the same purpose. Please replace either or use none :)

Reply:
I replaced the usage of deepmerge-ts with defu but noticed that deepmerge-ts seemed to pretty much ignore the types when mapping the storybook actions and argTypes. For now I just continued to use a type assertion to make the return type be "Meta" now just explicit instead of implicit.
For the future I'd suggest to maybe look into the utility function again and simplify it's options as the whole meta object can be passed in there only for two properties of it to be re-mapped and returned. But that's nothing related to this PR.